.github/workflows/ci.yml

```
@@ -0,0 +1,404 @@
+on:
+  pull_request:
+    types: [opened, synchronize, reopened]
+  push:
+    branches:
+      - master
+      - '0.[0-9]+.x'
+      - '1.[0-9]+.x'
+
+
+name: CI Tests
+
+jobs:
+  check_and_test:
+    name: Check
+    strategy:
+      fail-fast: false
+      matrix:
+        rust: ["stable", "beta", "nightly"]
+        backend: ["postgres", "sqlite", "mysql"]
+        os: [ubuntu-18.04, macos-latest, windows-latest]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - name: Checkout sources
+        uses: actions/checkout@v1
+
+      - name: Cache cargo registry
+        uses: actions/cache@v1
```

actions/cache v2 is also released now.

```suggestion
        uses: actions/cache@v2
```